packages/util/src/bytes.ts

```diff
@@ -1,8 +1,30 @@
 import BN from 'bn.js'
-import { intToBuffer, stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
+import { stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
 import { PrefixedHexString, TransformableToArray, TransformableToBuffer } from './types'
 import { assertIsBuffer, assertIsArray, assertIsHexString } from './helpers'
 
+/**
+ * Converts a `Number` into a hex `String`
+ * @param {Number} i
+ * @return {String}
+ */
+export const intToHex = function (i: number) {
```

Comment:
Not a big deal, but wouldn't it be simpler to just:
`export function intToHex(i: number) { ... ` ?

Reply:
Maybe, but this file exports all methods like this :smile: